pkg/apis/crd/v1alpha2/types.go

```diff
@@ -455,3 +455,95 @@ type TrafficControlList struct {
 
 	Items []TrafficControl `json:"items"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// BGPPolicy defines BGP configuration applied to Nodes.
+type BGPPolicy struct {
```

Comment:
Since this is a completely new feature / CRD, I don't see any reason to have it in `v1alpha2` instead of `v1alpha1`

Reply:
Got that. I will move it to `v1alpha1`.